jdaviz/app.py

```diff
@@ -1645,6 +1645,8 @@ def _create_viewer_item(self, viewer, vid=None, name=None, reference=None):
             'viewer_options': "IPY_MODEL_" + viewer.viewer_options.model_id,
             'selected_data_items': {},  # noqa data_id: visibility state (visible, hidden, mixed), READ-ONLY
             'visible_layers': {},  # label: {color, label_suffix}, READ-ONLY
+            'rotation': 0,  # canvas rotation clockwise rotation angle in deg
+            'flip': False,  # canvas rotation horizontal flip
```

Comment:
I feel this key name should be more specific in case someone asks for vertical flip in the future.
```suggestion
            'flip_horizontal': False,  # canvas rotation horizontal flip
```